CONTRIBUTORS.txt

```diff
@@ -14,18 +14,24 @@ needs to be listed here.
 - Adam Nemecek <adamnemecek@gmail.com>
 - Alex Blewitt <alex.blewitt@gmail.com>
 - Andrew Bennett <635596+therealbnut@users.noreply.github.com>
+- Andrew Edwards <andrewangeta@gmail.com>
 - Ankit Aggarwal <ankit_aggarwal@apple.com>
 - Arjun Nayini <aanayini@gmail.com>
 - Bas Broek <bas@basbroek.nl>
 - Ben Cohen <airspeedswift@users.noreply.github.com>
 - Cory Benfield <cbenfield@apple.com>
 - Damiaan Dufaux <damiaan@dpro.be>
 - Daniel Dunbar <daniel_dunbar@apple.com>
+- Esteban Torres <me@estebantorr.es>
+- Eugen <eugenf78@gmail.com>
 - Frank Kair <frankkair@gmail.com>
+- Franz Busch <privat@franz-busch.de>
 - Gopal Sharma <gopal@the-sharmas.org>
 - Helge Heß <helge@alwaysrightinstitute.com>
 - Ian Partridge <i.partridge@uk.ibm.com>
 - Jason Toffaletti <toffaletti@gmail.com>
+- Jim Dovey <jimdovey@mac.com>
+- Johannes Weiss <johannesweiss@apple.com>
```

Comment:
Let's remove the contributors script, as it has inadvertently added Johannes twice. We should re-run it in a separate PR, and then add it to our sanity check build system to start rejecting builds where the user is not in the contributor mesh.

Reply:
Ok, see #550 for this. You should leave your own name's entry here in the right place, as that will prevent us hitting a problem if we end up merging #550 before this patch.

Reply:
Hehe, I was going to suggest the `.mailmap` thing—I saw the same in the http2 repo, and that the `.mailmap` file there had two entries for @weissi that were the wrong way round, it seems. I copied his approach first, but then that didn't work so I copied yours 😉

Reply:
I've simply reverted the Contributors file changes; if the script will be run & inlined with the build or merge process then that should be handled by something other than my commit, I think. I expect Github provides something like a post-pull-request-approval hook that could be used to automate the process?

Reply:
Oh, wait, I mis-read. So I should update the contributors file but manually delete @weissi's ASCII clone—no problem.

Reply:
Sorry, no, you were right the first time: let's remove the entire CONTRIBUTORS file change except your entry.

Reply:
sorry guys 😬. I recently had too much trouble with 'Weiß' so switched to 'Weiss' everywhere 🙃.